### ISSUE TITLE
Speaker only recognizes once

### ISSUE THREAD

**Issue by mswellsi** [Azure-Samples/Cognitive-Services-Voice-Assistant#174]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> Use the speaker for two commands, eg computer turn the TV ON, computer turn the TV OFF
The second keyword is not recognized. The app needs to be restarted each time.

### Any log messages given by the failure
> None

### Expected/desired behavior
> Both utterances are recognized. 

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by mswellsi** [comment]
Using the released SDK (v1.10) and the released DDK (v1.9) we have the speaker only able to recognize one keyword at a time.

If I use the  .so files that Chris has, then KWS fully works.
Also KWS is working on RPi.

**Comment by mswellsi** [comment]
Using 1.11 of SDK and DDK the new error is
Cannot set number of channels for device "default": Invalid argument

**Comment by mswellsi** [comment]
v1.9 & v1.10 had the bug that caused this, it was fixed in v1.11.

For the previous comment the wrong v1.11 binaries were used. Using the 'speaker' binaries from the DDK it now works as expected.
The correct binaries will be available here https://aka.ms/sdsdk-download-speaker once they make it through the release process.
